src/assets/Keycloak/oCIS-with-Keycloak.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Integrating oCIS With Keycloak
+authorName: Amrita Shrestha
+authorAvatar: https://avatars.githubusercontent.com/u/54478846?s=100&v=4
+authorLink: https://github.com/amrita-shrestha
+createdAt: May 31, 2024
+tags: ownCloud, oCIS, oCIS-with-Keycloak, Keycloak
+banner: https://blog.jankaritech.com/src/assets/Keycloak/images/oCIS-Keycloak/oCIS-with-Keycloak.png
+---
+
+[In the previous blog](/src/assets/Keycloak/Keycloak.md), we explored the general functionality of Keycloak.
+In this blog, I demonstrate how we can integrate [oCIS](https://owncloud.dev/ocis/)(file-sync and share platform) with Keycloak using the Keycloak WebUI.
+Anyone interested in getting hands-on experience with Keycloak can read this blog.
```

Comment:
Here is that tiny suggestion I had mentioned in the last [review](https://github.com/JankariTech/blog/pull/122#pullrequestreview-2095839723). I don’t know where it had gone… I guess I made some mistake when doing the review.

The English modal verb “can” means “to be able to do something” or “to be allowed to do something” (https://dictionary.cambridge.org/dictionary/learner-english/can)

I assume what you wanted to say has nothing to do with the ability or permission of the reader but rather something like: 

```suggestion
This benefits anyone who is interested in getting hands-on experience with Keycloak.
```